client/src/EntryForm.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import Input from './Input';
+import CustomButton from './CustomButton';
+import ImageField from './ImageField';
+
+export default function EntryForm() {
+  const [location, setLocation] = useState('');
+  const [travelDate, setTravelDate] = useState('');
+  const [blurb, setBlurb] = useState('');
+  const [imageUrl, setImageUrl] = useState('');
+
+  const handleLocationChange = (e) => {
+    setLocation(e.target.value);
+  };
+
+  const handleTravelDateChange = (e) => {
+    setTravelDate(e.target.value);
+  };
+
+  const handleBlurbChange = (e) => {
+    setBlurb(e.target.value);
+  };
+
+  const handleImageUrlChange = (e) => {
+    setImageUrl(e.target.value);
+  };
+
+  const handleSubmit = async (e) => {
+    // Remove this preventDefault when done testing
```

Comment:
You will probably want the preventDefault to stay there forever. The default action on submit is to send a request to the backend and navigate to a new page. You don't want that since you are handling all the actions yourself.